### PR TITLE
[INLONG-4576][Manager] Use Junit5(Jupiter) to replace Junit4

### LINKED ISSUE #4576: [Improve][Manager] Upgrade Junit4 -> Junit5 Jupiter
### Description

Upgrade Junit4 -> Junit5 Jupiter 

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by leosanqing**
### Prepare a Pull Request

- Fixes #4576 

### Motivation
It's time to upgrade Junit version ,It provides more functions 

### Modifications
- Manager unit-tests 
- Maven dependencies

### CODE REVIEW

**Comment by leosanqing** [comment]
The current branch has code unrelated to the Junit upgrade, which will be corrected later

File: inlong-manager/manager-client/src/test/java/org/apache/inlong/manager/client/api/inner/InnerInlongManagerClientTest.java [hunk 1]
```diff
@@ -86,11 +86,11 @@ class InnerInlongManagerClientTest {
 
     @BeforeAll
     static void setup() {
-        wireMockServer = new WireMockServer(options().port(8084));
+        wireMockServer = new WireMockServer(options().port(8484));
```
Comment (healchow) [inline inlong-manager/manager-client/src/test/java/org/apache/inlong/manager/client/api/inner/InnerInlongManagerClientTest.java:89]: Why change the port?

Comment (healchow) [inline inlong-manager/manager-client/src/test/java/org/apache/inlong/manager/client/api/inner/InnerInlongManagerClientTest.java:89]: Why change the port?

File: inlong-manager/manager-client-examples/src/test/java/org/apache/inlong/manager/client/AutoPush2HiveExample.java [hunk 2]
```diff
@@ -29,18 +29,20 @@
...
 import java.util.concurrent.TimeUnit;
 
 /**
  * Test class for auto push to hive.
  */
-public class AutoPush2HiveExample extends BaseExample {
+@Disabled
+class AutoPush2HiveExample extends BaseExample {
 
     @Test
```
Comment (healchow) [inline inlong-manager/manager-client-examples/src/test/java/org/apache/inlong/manager/client/AutoPush2HiveExample.java:44]: If this is not the unit test class, suggest removing all `@Test` annotations.

**Review by healchow — CHANGES_REQUESTED** [review]

File: inlong-manager/manager-client/src/test/java/org/apache/inlong/manager/client/api/inner/InnerInlongManagerClientTest.java (hunk 1 above)
Comment (leosanqing) [reply inlong-manager/manager-client/src/test/java/org/apache/inlong/manager/client/api/inner/InnerInlongManagerClientTest.java]: This port has been used

File: inlong-manager/manager-client-examples/src/test/java/org/apache/inlong/manager/client/AutoPush2HiveExample.java (hunk 2 above)
Comment (leosanqing) [reply inlong-manager/manager-client-examples/src/test/java/org/apache/inlong/manager/client/AutoPush2HiveExample.java:44]: This port has been used

Comment (healchow) [reply inlong-manager/manager-client-examples/src/test/java/org/apache/inlong/manager/client/AutoPush2HiveExample.java:44]: OK, good idea.

**Review by healchow — APPROVED** [review]
LGTM

**Review by kipshi — APPROVED** [review]

File: inlong-manager/manager-client-examples/src/test/java/org/apache/inlong/manager/client/Kafka2HiveExample.java [hunk 3]
```diff
@@ -39,14 +40,15 @@
...
+@Disabled
 public class Kafka2HiveExample extends BaseExample {
 
     @Test
     public void testCreateGroupForHive() {
         ClientConfiguration configuration = new ClientConfiguration();
-        configuration.setWriteTimeout(10);
-        configuration.setReadTimeout(10);
-        configuration.setConnectTimeout(10);
+        configuration.setWriteTimeout(1000);
```
Comment (healchow) [inline inlong-manager/manager-client-examples/src/test/java/org/apache/inlong/manager/client/Kafka2HiveExample.java:49]: Why increase those timeouts?

**Review by healchow — CHANGES_REQUESTED** [review]

**Review by woofyzhao — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**